app/views/admin/pages/index.html.erb

```diff
@@ -2,14 +2,50 @@
 
 <header class="flex items-center mb-6">
   <h2 class="crayons-title">Pages</h2>
-  <%= link_to "New page", new_admin_page_path, class: "btn btn-primary ml-auto" %>
+  <%= link_to "New page", new_admin_page_path, class: "ml-auto crayons-btn crayons-btn--s" %>
 </header>
 
-<div class="crayons-card p-6">
+<div class="crayons-card p-6 pages__table">
   <% @pages.each do |page| %>
     <div class="flex py-2 items-center">
-      <%= link_to page.title, edit_admin_page_path(page.id) %>
-      <%= link_to "View", page.path, class: "ml-auto btn btn-secondary" %>
+      <%= link_to page.title, page.path %>
+      <%= link_to "Edit", edit_admin_page_path(page.id), class: "ml-auto crayons-btn crayons-btn--s crayons-btn--secondary" %>
     </div>
   <% end %>
 </div>
+
+<div class="mt-8 pages__override_defaults">
+  <div class="my-6">
+    <% if !@code_of_conduct || !@privacy || !@terms%>
+      <h2 class="mb-4 crayons-title">Override defaults</h2>
+      <div class="crayons-notice crayons-notice--warning">
+        <strong>Note: Proceed with caution. When you edit any of the following pages, it will diverge from the original Forem version and you will no longer receive updates. You will see your updated version in the section above.</strong>
+      </div>
+    <% else %>
+      <div class="crayons-notice crayons-notice--info">
+        <strong>Note: You have overridden the code of conduct, privacy policy and terms of use. You will no longer receive updates on these pages from Forem.</strong>
+      </div>
+    <% end %>
```

Comment:
If the user has already overridden all three items then we jsut display that information to them